service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ImmunizationObservationStatementMapper.java

```diff
@@ -52,7 +52,7 @@
     private static final String QUANTITY = "Quantity: ";
     private static final String REASON = "Reason: ";
     private static final String REASON_NOT_GIVEN = "Reason not given: ";
-    private static final String VACCINATION_PROTOCOL_STRING = "Vaccination Protocol %S: %s Sequence: %S,%S ";
+    private static final String VACCINATION_PROTOCOL_STRING = "Vaccination Protocol %S: %s Sequence: %S, Doses: %S ";
```

Comment:
Better to call those `SOMETHING_TEMPLATE` like `VACCINATION_PROTOCOL_TEMPLATE`

Reply:
Use lower case `%s`

Reply:
Addressed